api/filters.go

```diff
@@ -178,16 +179,20 @@ func (api *API) submitFilter(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	var dIDs []string
+	for _, d := range filter.Dimensions {
+		dIDs = append(dIDs, d.Name)
```

Comment:
Are we using dimension names or dimension IDs? We need to be clear and keep the naming accurate